tests/transport_test.go

```diff
@@ -48,20 +62,19 @@ var _ = Describe("Transport Tests", func() {
 
 	// it() is the body of the test and is executed once per Entry() by DescribeTable()
 	// closes over c and ns
-	it := func(ep string, credentialed bool) {
-
-		pvcAnn := map[string]string{
-			controller.AnnEndpoint: ep,
-			controller.AnnSecret:   "",
-		}
-
+	it := func(ip, file, accessKey, secretKey string) {
 		var (
 			err error // prevent shadowing
 			sec *v1.Secret
 		)
 
-		if credentialed {
-			By(fmt.Sprintf("Creating secret for endpoint %s", ep))
+		pvcAnn := map[string]string{
+			controller.AnnEndpoint: ip + "/" + file,
+			controller.AnnSecret:   "",
+		}
+
+		if accessKey != "" || secretKey != "" {
```

Comment:
Just reviewed this pattern [here](https://github.com/kubevirt/containerized-data-importer/pull/427/files#diff-11ace0428af6f34ce6ac5652673678beR66). Is there some leverage?

Reply:
We're changing the same line, so the code isn't going to be duplicated.